docs/server.rst

```diff
@@ -54,6 +54,11 @@ The following modules are required for Tunneldigger operation:
 * ``l2tp_eth``
 * ``l2tp_netlink``
 
+.. note::
+
+    Fedora and RHEL/CentOS blacklist some of these modules by default for security reasons.
+    Ensure you remove the blacklisting from ``/etc/modprobe.d/l2tp_eth-blacklist.conf`` and ``/etc/modprobe.d/l2tp_netlink-blacklist.conf``.
```

Comment:
Good catch! This could have caused quite the headaches...

Reply:
It did already. For me :laughing:

Reply:
Thanks for avoiding this for the rest of us :D